### PR TITLE
doc review

### PR DESCRIPTION
**Pull request by dgarus**
## Description

Part of #1319 

- fixed wrong enumeration
- fixed wrong link to the contribution guide
- extended description how to install `protoc`
- added a few links

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/community/get_involved/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).

### Code Contributions

- [x] I've run [pre-commit.sh](https://github.com/pyrsia/pyrsia/blob/main/pre-commit.sh) ([pre-commit.bat](https://github.com/pyrsia/pyrsia/blob/main/pre-commit.bat)) successfully.
- [x] I've run the [integrations tests](https://github.com/pyrsia/pyrsia-integration-tests#how-to-set-up-and-run-the-tests)
on the PR branch and all tests passes.

### CODE REVIEW

**Review by erwin1 — DISMISSED** [review]

File: docs/community/get_involved/local_dev_setup.md [hunk 1]
```diff
@@ -174,15 +203,15 @@ You can use the Pyrsia CLI to ensure that the peers are connected.
     Connected Peers Count:       1
     ```
 
-If you see a status message similar to:
+    If you see a status message similar to:
```
Comment (betarelease) [inline docs/community/get_involved/local_dev_setup.md:206]: The linter may fail these due to tabs/spaces showing up. You may want to run markdown linter locally first and update this PR.

Comment (betarelease) [reply docs/community/get_involved/local_dev_setup.md:206]: Here are the linting rules we use 

https://github.com/pyrsia/pyrsia/blob/main/.github/linters/.markdownlint.yaml


I have the `markdownlint` extension in my VS Code - it does all the linting for me in the editor.

**Review by karolh2000 — APPROVED** [review]

**Review by dasmanas — DISMISSED** [review]

File: docs/community/get_involved/local_dev_setup.md (hunk 1 above)
Comment (dgarus) [reply docs/community/get_involved/local_dev_setup.md]: Thanks. Fixed

**Review by fishseabowl — APPROVED** [review]

**Review by sbtaylor15 — APPROVED** [review]
/lgtm

**Review by karolh2000 — APPROVED** [review]

**Pull request merged**